frontend/src/container/OnboardingContainer/constants/apmDocFilePaths.ts

```diff
@@ -1053,6 +1054,7 @@ export const ApmDocFilePaths = {
 	APM_java_springBoot_kubernetes_recommendedSteps_setupOtelCollector,
 	APM_java_springBoot_kubernetes_recommendedSteps_instrumentApplication,
 	APM_java_springBoot_kubernetes_recommendedSteps_runApplication,
+	APM_java_springBoot_kubernetes_recommendedSteps_runApplication_producer,
```

Comment:
Will be adding all others `.md` changes for other cross combination of `lang x datasource x kafka services` in coming PRs